nlp-architect fails to build on macOS 10.14 Mojave #195

During pip install -e . , It gives following error. Seems like clang/gcc is pretty broken.

Last login: Wed May 22 10:51:01 on console
Prabhats-MacBook-Air:~ pksingh$ cd nlp-architect/
Prabhats-MacBook-Air:nlp-architect pksingh$ pip install -e .
Obtaining file:///Users/pksingh/nlp-architect
Requirement already satisfied: tensorflow==1.13.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (1.13.1)
Collecting dynet==2.0.2 (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/01/10/cfc00197733dd7cf52d9e00c42017ebe7e818653980bccfe6f241ea5b79a/dyNET-2.0.2.tar.gz
Collecting spacy (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/80/d2/61774b69cd79abbf5de91af0093f8cc919f4b28849787982daaa449fbf5f/spacy-2.1.4-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting nltk (from nlp-architect==0.4.post1)
Collecting gensim (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/82/bb/56f295a604dfafdef746cc81081ff4c6e825690de95963000300a1cd3d80/gensim-3.7.3-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting sklearn (from nlp-architect==0.4.post1)
Requirement already satisfied: scipy in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (1.2.0)
Requirement already satisfied: numpy in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (1.16.0)
Collecting tensorflow_hub (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/10/5c/6f3698513cf1cd730a5ea66aec665d213adf9de59b34f362f270e0bd126f/tensorflow_hub-0.4.0-py2.py3-none-any.whl
Collecting elasticsearch (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/82/c9/3920effd37e555f670bc522483dc940eb4197b9a4d3d95dd2a05842be849/elasticsearch-7.0.1-py2.py3-none-any.whl
Collecting fasttextmirror (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/fb/78/cf79876cfbb92bf7baae65472b19c680f6e20eaf55ca41721a53ea2014bb/fasttextmirror-0.8.22.tar.gz
Collecting newspaper3k (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/d7/b9/51afecb35bb61b188a4b44868001de348a0e8134b4dfa00ffc191567c4b9/newspaper3k-0.2.8-py3-none-any.whl
Collecting wordfreq (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/d0/82/233c39f350ac66c740266dafb348f9e67ba3a5e5dad6a949a2c3715c34f5/wordfreq-2.2.1-py3-none-any.whl
Collecting seqeval (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/55/dd/3bf1c646c310daabae47fceb84ea9ab66df7f518a31a89955290d82b8100/seqeval-0.0.10-py3-none-any.whl
Collecting pywikibot (from nlp-architect==0.4.post1)
Collecting num2words (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/eb/a2/ea800689730732e27711c41beed4b2a129b34974435bdc450377ec407738/num2words-0.5.10-py3-none-any.whl
Collecting hyperopt (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/63/12/704382c3081df3ae3f9d96fe6afb62efa2fa9749be20c301cd2797fb0b52/hyperopt-0.1.2-py3-none-any.whl
Requirement already satisfied: h5py in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (2.9.0)
Collecting pandas (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/fc/43/fd867e3347559845c8f993059d410c50a1e18709f1c4d4b3b47323a06a37/pandas-0.24.2-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Requirement already satisfied: tqdm in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (4.29.1)
Collecting ftfy (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/8f/86/df789c5834f15ae1ca53a8d4c1fc4788676c2e32112f6a786f2625d9c6e6/ftfy-5.5.1-py3-none-any.whl
Collecting bokeh (from nlp-architect==0.4.post1)
Requirement already satisfied: six in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (1.12.0)
Requirement already satisfied: future in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (0.17.1)
Requirement already satisfied: requests in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (2.20.0)
Requirement already satisfied: termcolor in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from nlp-architect==0.4.post1) (1.1.0)
Collecting pillow (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/22/55/2ce41fa510f131c776112a1d24ee90cddffc96f1bf0311efb14fdd8ae877/Pillow-6.0.0-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting hug (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/04/d9/1b0cf6bd3cd1cb8b33246c48105bc1cbf87108c35d1e444a325460a32d56/hug-2.5.4-py2.py3-none-any.whl
Collecting falcon==1.4.1 (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/e8/d0/20bb807dee65f1f163754670557b128eafce1710f6c9c363a38e357f3783/falcon-1.4.1-py2.py3-none-any.whl
Collecting falcon_multipart (from nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/e1/a2/e50ffc3101ed6b91d1edc63b3586c424ef8071e1ef0ef7dcb8745e65fc14/falcon_multipart-0.2.0-py3-none-any.whl
Requirement already satisfied: grpcio>=1.8.6 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (1.20.0)
Requirement already satisfied: gast>=0.2.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (0.2.2)
Requirement already satisfied: wheel>=0.26 in /Users/pksingh/Library/Python/3.7/lib/python/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (0.33.1)
Requirement already satisfied: tensorflow-estimator<1.14.0rc0,>=1.13.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (1.13.0)
Requirement already satisfied: astor>=0.6.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (0.7.1)
Requirement already satisfied: protobuf>=3.6.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (3.7.1)
Requirement already satisfied: keras-preprocessing>=1.0.5 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (1.0.9)
Requirement already satisfied: tensorboard<1.14.0,>=1.13.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (1.13.1)
Requirement already satisfied: keras-applications>=1.0.6 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (1.0.7)
Requirement already satisfied: absl-py>=0.1.6 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow==1.13.1->nlp-architect==0.4.post1) (0.7.1)
Requirement already satisfied: cython in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from dynet==2.0.2->nlp-architect==0.4.post1) (0.29.7)
Collecting preshed<2.1.0,>=2.0.1 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/48/fe/2f2e8c91541785f2abe0d51f37eb00356513b9ff3d24fb27fd5b59e18264/preshed-2.0.1-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting plac<1.0.0,>=0.9.6 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/9e/9b/62c60d2f5bc135d2aa1d8c8a86aaf84edb719a59c7f11a4316259e61a298/plac-0.9.6-py2.py3-none-any.whl
Requirement already satisfied: jsonschema<3.1.0,>=2.6.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from spacy->nlp-architect==0.4.post1) (2.6.0)
Collecting wasabi<1.1.0,>=0.2.0 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/f4/c1/d76ccdd12c716be79162d934fe7de4ac8a318b9302864716dde940641a79/wasabi-0.2.2-py3-none-any.whl
Collecting murmurhash<1.1.0,>=0.28.0 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/b9/bd/faace403086ee922afc74e5615cb8c21020fcf5d5667314e943c08f71fde/murmurhash-1.0.2-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting srsly<1.1.0,>=0.0.5 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/c8/b2/d2cc9f5aa5a458aca45d8689279b41d4b42ba4e8c63b0cb6a9f009340fd0/srsly-0.0.5-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting blis<0.3.0,>=0.2.2 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/36/41/9e934e2b8a2cdae447ed1923a94f98c2d70c898b65af6635f5fe55f7ed4d/blis-0.2.4-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting thinc<7.1.0,>=7.0.2 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/92/b1/d7df83813ee3c42d46e6ddf4d4f1d9bd35a4735827d35b7f02539bea3136/thinc-7.0.4-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting cymem<2.1.0,>=2.0.2 (from spacy->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/d7/11/37da628920bf2999bd8c4ffc40908413622486d5dbc4e60d87a58c428367/cymem-2.0.2-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting smart-open>=1.7.0 (from gensim->nlp-architect==0.4.post1)
Requirement already satisfied: scikit-learn in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from sklearn->nlp-architect==0.4.post1) (0.20.2)
Requirement already satisfied: urllib3>=1.21.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from elasticsearch->nlp-architect==0.4.post1) (1.24.2)
Collecting pybind11>=2.2 (from fasttextmirror->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/f2/7c/e71995e59e108799800cb0fce6c4b4927914d7eada0723dd20bae3b51786/pybind11-2.2.4-py2.py3-none-any.whl
Requirement already satisfied: setuptools>=0.7.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from fasttextmirror->nlp-architect==0.4.post1) (40.8.0)
Collecting beautifulsoup4>=4.4.1 (from newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/1d/5d/3260694a59df0ec52f8b4883f5d23b130bc237602a1411fa670eae12351e/beautifulsoup4-4.7.1-py3-none-any.whl
Collecting feedparser>=5.2.1 (from newspaper3k->nlp-architect==0.4.post1)
Requirement already satisfied: PyYAML>=3.11 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from newspaper3k->nlp-architect==0.4.post1) (5.1)
Collecting tldextract>=2.0.1 (from newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/1e/90/18ac0e5340b6228c25cc8e79835c3811e7553b2b9ae87296dfeb62b7866d/tldextract-2.2.1-py2.py3-none-any.whl
Collecting jieba3k>=0.35.1 (from newspaper3k->nlp-architect==0.4.post1)
Collecting tinysegmenter==0.3 (from newspaper3k->nlp-architect==0.4.post1)
Collecting feedfinder2>=0.0.4 (from newspaper3k->nlp-architect==0.4.post1)
Collecting lxml>=3.6.0 (from newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/45/6c/436a534dca42f7982ba793983353035d117ab70541266704974efa323ade/lxml-4.3.3-cp37-cp37m-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Collecting cssselect>=0.9.2 (from newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/7b/44/25b7283e50585f0b4156960691d951b05d061abf4a714078393e51929b30/cssselect-1.0.3-py2.py3-none-any.whl
Requirement already satisfied: python-dateutil>=2.5.3 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from newspaper3k->nlp-architect==0.4.post1) (2.7.5)
Collecting regex<=2018.02.21,>=2017.07.11 (from wordfreq->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/a2/51/c39562cfed3272592c60cfd229e5464d715b78537e332eac2b695422dc49/regex-2018.02.21.tar.gz
Collecting langcodes>=1.4.1 (from wordfreq->nlp-architect==0.4.post1)
Collecting msgpack (from wordfreq->nlp-architect==0.4.post1)
Collecting Keras>=2.2.4 (from seqeval->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/5e/10/aa32dad071ce52b5502266b5c659451cfd6ffcbf14e6c8c4f16c0ff5aaab/Keras-2.2.4-py2.py3-none-any.whl
Requirement already satisfied: docopt>=0.6.2 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from num2words->nlp-architect==0.4.post1) (0.6.2)
Requirement already satisfied: pymongo in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from hyperopt->nlp-architect==0.4.post1) (3.7.2)
Requirement already satisfied: networkx in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from hyperopt->nlp-architect==0.4.post1) (2.2)
Requirement already satisfied: pytz>=2011k in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from pandas->nlp-architect==0.4.post1) (2018.9)
Requirement already satisfied: wcwidth in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from ftfy->nlp-architect==0.4.post1) (0.1.7)
Collecting tornado>=4.3 (from bokeh->nlp-architect==0.4.post1)
Requirement already satisfied: packaging>=16.8 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from bokeh->nlp-architect==0.4.post1) (18.0)
Requirement already satisfied: Jinja2>=2.7 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from bokeh->nlp-architect==0.4.post1) (2.10.1)
Requirement already satisfied: certifi>=2017.4.17 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from requests->nlp-architect==0.4.post1) (2019.3.9)
Requirement already satisfied: idna<2.8,>=2.5 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from requests->nlp-architect==0.4.post1) (2.7)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from requests->nlp-architect==0.4.post1) (3.0.4)
Collecting python-mimeparse>=1.5.2 (from falcon==1.4.1->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/26/2e/03bce213a9bf02a2750dcb04e761785e9c763fc11071edc4b447eacbb842/python_mimeparse-1.6.0-py2.py3-none-any.whl
Requirement already satisfied: mock>=2.0.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorflow-estimator<1.14.0rc0,>=1.13.0->tensorflow==1.13.1->nlp-architect==0.4.post1) (2.0.0)
Requirement already satisfied: markdown>=2.6.8 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorboard<1.14.0,>=1.13.0->tensorflow==1.13.1->nlp-architect==0.4.post1) (3.1)
Requirement already satisfied: werkzeug>=0.11.15 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from tensorboard<1.14.0,>=1.13.0->tensorflow==1.13.1->nlp-architect==0.4.post1) (0.15.2)
Collecting boto>=2.32 (from smart-open>=1.7.0->gensim->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/23/10/c0b78c27298029e4454a472a1919bde20cb182dab1662cec7f2ca1dcc523/boto-2.49.0-py2.py3-none-any.whl
Requirement already satisfied: boto3 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from smart-open>=1.7.0->gensim->nlp-architect==0.4.post1) (1.9.96)
Collecting soupsieve>=1.2 (from beautifulsoup4>=4.4.1->newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/b9/a5/7ea40d0f8676bde6e464a6435a48bc5db09b1a8f4f06d41dd997b8f3c616/soupsieve-1.9.1-py2.py3-none-any.whl
Collecting requests-file>=1.4 (from tldextract>=2.0.1->newspaper3k->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/23/9c/6e63c23c39e53d3df41c77a3d05a49a42c4e1383a6d2a5e3233161b89dbf/requests_file-1.4.3-py2.py3-none-any.whl
Collecting marisa-trie (from langcodes>=1.4.1->wordfreq->nlp-architect==0.4.post1)
  Using cached https://files.pythonhosted.org/packages/20/95/d23071d0992dabcb61c948fb118a90683193befc88c23e745b050a29e7db/marisa-trie-0.7.5.tar.gz
Requirement already satisfied: decorator>=4.3.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from networkx->hyperopt->nlp-architect==0.4.post1) (4.4.0)
Requirement already satisfied: pyparsing>=2.0.2 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from packaging>=16.8->bokeh->nlp-architect==0.4.post1) (2.4.0)
Requirement already satisfied: MarkupSafe>=0.23 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from Jinja2>=2.7->bokeh->nlp-architect==0.4.post1) (1.1.1)
Requirement already satisfied: pbr>=0.11 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from mock>=2.0.0->tensorflow-estimator<1.14.0rc0,>=1.13.0->tensorflow==1.13.1->nlp-architect==0.4.post1) (5.1.3)
Requirement already satisfied: jmespath<1.0.0,>=0.7.1 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from boto3->smart-open>=1.7.0->gensim->nlp-architect==0.4.post1) (0.9.4)
Requirement already satisfied: s3transfer<0.3.0,>=0.2.0 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from boto3->smart-open>=1.7.0->gensim->nlp-architect==0.4.post1) (0.2.0)
Requirement already satisfied: botocore<1.13.0,>=1.12.96 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from boto3->smart-open>=1.7.0->gensim->nlp-architect==0.4.post1) (1.12.134)
Requirement already satisfied: docutils>=0.10 in /Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages (from botocore<1.13.0,>=1.12.96->boto3->smart-open>=1.7.0->gensim->nlp-architect==0.4.post1) (0.14)
Building wheels for collected packages: dynet, fasttextmirror, regex, marisa-trie
  Building wheel for dynet (setup.py) ... error
  ERROR: Complete output from command /Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7 -u -c 'import setuptools, tokenize;__file__='"'"'/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-wheel-v0wl9pe8 --python-tag cp37:
  ERROR: running bdist_wheel
  running build
  INFO:root:CMAKE_PATH='/usr/local/bin/cmake'
  INFO:root:MAKE_PATH='/usr/bin/make'
  INFO:root:MAKE_FLAGS='-j 4'
  INFO:root:EIGEN3_INCLUDE_DIR='/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/eigen'
  INFO:root:EIGEN3_DOWNLOAD_URL='https://bitbucket.org/eigen/eigen/get/699b6595fc47.zip'
  INFO:root:CC_PATH='/usr/bin/gcc'
  INFO:root:CXX_PATH='/usr/bin/g++'
  INFO:root:SCRIPT_DIR='/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet'
  INFO:root:BUILD_DIR='/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit'
  INFO:root:INSTALL_PREFIX='/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/../../..'
  INFO:root:PYTHON='/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7'
  cmake version 3.14.4
  
  CMake suite maintained and supported by Kitware (kitware.com/cmake).
  Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk/usr/include/c++/4.2.1
  Apple LLVM version 10.0.1 (clang-1001.0.46.4)
  Target: x86_64-apple-darwin18.6.0
  Thread model: posix
  InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
  INFO:root:Creating build directory /private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit
  INFO:root:Fetching Eigen...
  INFO:root:Unpacking Eigen...
  INFO:root:Configuring...
  -- The C compiler identification is AppleClang 10.0.1.10010046
  -- The CXX compiler identification is AppleClang 10.0.1.10010046
  -- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/usr/bin/gcc
  -- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/usr/bin/gcc -- broken
  CMake Error at /usr/local/Cellar/cmake/3.14.4/share/cmake/Modules/CMakeTestCCompiler.cmake:60 (message):
    The C compiler
  
      "/Applications/Xcode.app/Contents/Developer/usr/bin/gcc"
  
    is not able to compile a simple test program.
  
    It fails with the following output:
  
      Change Dir: /private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/CMakeFiles/CMakeTmp
  
      Run Build Command(s):/usr/bin/make cmTC_9f760/fast
      /Applications/Xcode.app/Contents/Developer/usr/bin/make -f CMakeFiles/cmTC_9f760.dir/build.make CMakeFiles/cmTC_9f760.dir/build
      Building C object CMakeFiles/cmTC_9f760.dir/testCCompiler.c.o
      /Applications/Xcode.app/Contents/Developer/usr/bin/gcc   -g -O2 -mmacosx-version-min=10.13 -isysroot /SDKs/MacOSX.platform/MacOSX10.13.sdk  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk   -o CMakeFiles/cmTC_9f760.dir/testCCompiler.c.o   -c /private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/CMakeFiles/CMakeTmp/testCCompiler.c
      Linking C executable cmTC_9f760
      /usr/local/Cellar/cmake/3.14.4/bin/cmake -E cmake_link_script CMakeFiles/cmTC_9f760.dir/link.txt --verbose=1
      /Applications/Xcode.app/Contents/Developer/usr/bin/gcc -g -O2 -mmacosx-version-min=10.13 -isysroot /SDKs/MacOSX.platform/MacOSX10.13.sdk  -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.14.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -mmacosx-version-min=10.13 -isysroot /SDKs/MacOSX.platform/MacOSX10.13.sdk  CMakeFiles/cmTC_9f760.dir/testCCompiler.c.o  -o cmTC_9f760
      clang: warning: no such sysroot directory: '/SDKs/MacOSX.platform/MacOSX10.13.sdk' [-Wmissing-sysroot]
      ld: library not found for -lSystem
      clang: error: linker command failed with exit code 1 (use -v to see invocation)
      make[1]: *** [cmTC_9f760] Error 1
      make: *** [cmTC_9f760/fast] Error 2
  
  
  
  
    CMake will not be able to correctly generate this project.
  Call Stack (most recent call first):
    CMakeLists.txt:1 (project)
  
  
  -- Configuring incomplete, errors occurred!
  See also "/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/CMakeFiles/CMakeOutput.log".
  See also "/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/CMakeFiles/CMakeError.log".
  error: /usr/local/bin/cmake /private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet -DCMAKE_INSTALL_PREFIX='/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/../../..' -DEIGEN3_INCLUDE_DIR='/private/var/folders/d5/_y4xg4k91b5gllsppplwsd5m0000gn/T/pip-install-rkn7ci4c/dynet/build/py3.7-64bit/eigen' -DPYTHON='/Library/Frameworks/Python.framework/Versions/3.7/bin/python3.7'

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 tensorflow==1.13.1
-dynet==2.0.2
+dynet==2.1
 spacy
 nltk
 gensim

